### PR TITLE
Add link to the drained assets reversal app to the homepage

### PR DESCRIPTION
**Pull request by yinkakun**

### CODE REVIEW

**Review by samgos — APPROVED** [review]
Perfect!

File: src/components/routes/Splash.tsx [hunk 1]
```diff
@@ -65,19 +65,64 @@ export default function Splash() {
...
+              and{" "}
+              <a
+                href="https://etherscan.io/token/0x17ac188e09a7890a1844e5e65471fe8b0ccfadf3"
+                target="_blank"
+                rel="noreferrer"
+              >
+                CC10
+              </a>{" "}
+              <a
+                href="https://discord.com/channels/@me/991236780676349963/994954651835314177"
```
Comment (samgos) [inline src/components/routes/Splash.tsx:92]: Incorrect href link.

Comment (yinkakun) [reply src/components/routes/Splash.tsx:92]: What should the correct href link be?

**Comment by samgos** [comment]
Fixed href for the post mortem, added FFF to the effected assets and explained that; only holders at the time of the hack and still maintain that balance are viable for redemption.

**Pull request merged**